code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -20,3 +20,19 @@
 	human.hairstyle = "Business Hair"
 	human.hair_color = "#bb9966" // brown
 	human.update_hair()
+
+/datum/species/human/get_scream_sound(mob/living/carbon/human/human)
```

Comment:
i think this should be the default definition of `get_scream_sound` as a fallback for all those species that don't have scream sounds.

Reply:
I'll leave that to a maintainer's decision. As it is right now - species not mentioned in the scream emote's proc won't make any sound while screaming. Unless a maintainer wants me to change it, that functionality is being kept the way it is, as my goal was to simply improve the code, without changing the surface of how it plays out

Reply:
You got it right